ibis/conftest.py

```diff
@@ -14,6 +14,7 @@ def add_ibis(monkeypatch, doctest_namespace):
     # reset interactive mode to False for doctests that don't execute
     # expressions
     ibis.options.interactive = False
+    ibis.options.repr.interactive.console_width = 80
```

Comment:
This seems to be setting the option for all the tests, so this os probably not the right place to set it. Although if I don't set it the doctests fail for me.